spec/unit/action_cable_client_spec.rb

```diff
@@ -108,7 +108,7 @@
           callback_called = true
         end
 
-        expect(@client).to receive(:_subscribed_callaback).twice.and_call_original
+        expect(@client).to receive(:_subscribed_callaback).and_call_original
```

Comment:
I'm not sure what the implications of this are, but I *think* this is a good change, and I'm glad it passes